Sources/Routing/Protocols/NavigationPathManageable.swift

```diff
@@ -10,28 +10,27 @@ import SwiftUI
 public protocol NavigationPathManageable: ObservableObject {
     associatedtype Destination: ViewDisplayable
 
-    var path: NavigationPath { get set }
+    var path: [Destination] { get set }
```

Comment:
```suggestion
    var stack: [Destination] { get set }
```

Would it make sense to rename this if it's no longer a NavigationPath?